lib/tasks/publish_deals.rake

```diff
@@ -0,0 +1,20 @@
+task :publish_deals => :environment do
```

Comment:
@jynti Add namespace and description to a rake task. This task is doing more than publishing deals.